db/schema.rb

```diff
@@ -90,9 +87,10 @@
     t.boolean  "has_partners",       limit: 1
     t.boolean  "has_scoreboard",     limit: 1
     t.boolean  "has_svn",            limit: 1
-    t.string   "remote_handin_path", limit: 255
     t.boolean  "quiz",               limit: 1,     default: false
     t.text     "quizData",           limit: 65535
+    t.string   "remote_handin_path", limit: 255
+    t.string   "category_name",      limit: 255
```

Comment:
I think it's good to be more explicit. (also makes it easy to refactor/debug for me)